docs/HARDWARE-VERIFICATION.md

```diff
@@ -0,0 +1,71 @@
+# Verifying EVE on edge nodes
+
+This is a specialized version of EVE-OS installer, that installs EVE-OS on the
+underlying edge node and verifies its compatibility. It checks if the necessary
+drivers are present, tests a static and dynamic networking configurations,
+tests the available storage devices, and tests if EVE-OS can spawn a guest
+successfully. Verification currently supports verification on x86 edge nodes.
```

Comment:
maybe the following is less redundant: "Verification is currently supported on x86 edge nodes"